projects/tokens/hardhat.config.ts

```diff
@@ -77,6 +77,7 @@ const config: HardhatUserConfig = {
   },
   paths: {
     sources: "contracts",
+    tests: "test-hardhat"
```

Comment:
Scoped hardhat to run test-hardhat to avoid it running the truffle tests, because hardhat-truffle5 doesn't have full compatibility with upgrades and was breaking.
Use truffle test for those until we rewrite them on hardhat.